backend/queries/events.js

```diff
@@ -359,56 +359,72 @@ const getDashEventsForVolunteer = async (volunteerId) => {
     GROUP BY volunteers.v_id, event_id, cohort_id
     ORDER BY event_start ASC;
   `;
-  return db.multi(importantsQuery + upcomingsQuery + pastsQuery, volunteerId)
-    .then(([importants, upcomings, pasts]) => {
-      return { importants, upcomings, pasts };
+
+  const pastYearData = `
+    SELECT
+        TO_CHAR(event_end, 'YYYY-MM') AS date, 
+	      COUNT(EXTRACT(MONTH FROM event_end)) AS events_count,
+	      SUM(event_volunteers.volunteered_time) AS hours
+        
+	    FROM volunteers
+	    INNER JOIN event_volunteers ON volunteer_id = v_id
+	    INNER JOIN events ON eventv_id = event_id
+	    WHERE event_end > (current_date - INTERVAL '12 months') 
+	    	AND event_end < CURRENT_DATE AND volunteers.v_id = 10 
```

Comment:
hard coded volunteers.v_id = 10?

Reply:
Thanks for catching that, I usually test my queries on Postico to make sire I have desired result and since it doesn’t handle variables I had to hard code it then I forgot to switch it back to a variable.